nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/io/marshallers/json/document/DocumentModelJsonReaderTest.java

```diff
@@ -167,4 +168,30 @@ public void testMultiSetOnNullDontSetDefaultValueAgain() throws Exception {
         assertNull(values.get("dv:multiWithDefault"));
     }
 
+    @Test
+    public void testReadContextData() throws Exception {
+        String noteJson = "{" //
+                + "\"entity-type\": \"document\"," //
+                + "\"type\": \"DocDefaultValue\"," //
+                + "\"name\": \"aDoc\"," //
+                + "\"contextData\": {"//
+                + "  \"request/comment\": \"a comment\"," //
```

Comment:
We don't use scopes in context data since 9.1 (https://jira.nuxeo.com/browse/NXP-21570) so please just use "comment".